base/src/main/java/org/aya/tyck/ExprTycker.java

```diff
@@ -467,6 +466,20 @@ private TermResult checkBoundaries(Expr expr, FormTerm.Path path, Subst subst, T
     });
   }
 
+  private FormTerm.Sort calculateSigma(@NotNull FormTerm.Sort x, @NotNull FormTerm.Sort y) {
```

Comment:
max

Reply:
It is not max anymore. It calculates possible sort for a sigma, which is sometimes not the max

Reply:
Does it also work for pi?

Reply:
no

Reply:
Why

Reply:
There is sortPi() for pi

Reply:
, which is adapted from agda

Reply:
Cool